paging/.idea/scopes/buildSrc.xml

```diff
@@ -0,0 +1 @@
+../../../.idea/scopes/buildSrc.xml
```

Comment:
these seems undesired or did you re-run the setup script and these are now tracked in .git while they werent before?

Reply:
ah thanks for catching this - i ran the script and I think I forgot to setup gitignore?

Reply:
Actually these were added by script, but I don't think this particular file should be here

Reply:
These files should be getting ignored by .gitignore but I'm getting:
```
~/p/a/paging (dlam/paging-playground|:) git ls-files -v ../.idea
H ../.idea/codeStyles/Project.xml
H ../.idea/codeStyles/codeStyleConfig.xml
H ../.idea/copyright/AndroidCopyright.xml
H ../.idea/copyright/profiles_settings.xml
H ../.idea/inspectionProfiles/Project_Default.xml
H ../.idea/scopes/Compose.xml
H ../.idea/scopes/Ignore_API_Files.xml
H ../.idea/scopes/buildSrc.xml
H ../.idea/vcs.xml
```

If I touch normal empty files, gitignore ignores them successfully, this is a bit puzzling :|

Reply:
```
~/p/a/paging (dlam/paging-playground|:) git ls-files -i ../.idea/* --exclude-standard
../.idea/inspectionProfiles/Project_Default.xml
../.idea/scopes/Compose.xml
../.idea/scopes/Ignore_API_Files.xml
../.idea/scopes/buildSrc.xml
```

Reply:
I'm tempted to say we should keep them because changes to those files actually do show up and get checked in even there's no explicit exclusion rule to allow them to not be ignored in .gitignore. bit confused here, but changes to the original files are being tracked.